content/en/users/security/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Security Services
+linkTitle: Security
+type: docs
+draft: true
+weight: 60
+description: >
+  Security services in the EGI Cloud
+---
+
+## Using cloud services securely
+
+Cloud security in the EGI Federated Cloud is the highest priority.
```

Comment:
I think we should not refer only to cloud here, as the service could be used also by other services in the EGI infra

Reply:
Fixed